lib/helpbase.gi

```diff
@@ -1172,6 +1172,11 @@ BindGlobal("HELP_TOPIC_RING", ListWithIdenticalEntries( HELP_RING_SIZE,
 if IsHPCGAP then
   LockAndMigrateObj(HELP_TOPIC_RING,HELP_REGION);
 fi;
+BindGlobal("HELP_ORIG_TOPIC_RING", ListWithIdenticalEntries( HELP_RING_SIZE,
+                                             "welcome to gap" ));
+if IsHPCGAP then
+  LockAndMigrateObj(HELP_ORIG_TOPIC_RING,HELP_REGION);
+fi;
```

Comment:
Please apply the same changes to `hpcgap/lib/helpbase.gi` (can be done relatively pain-free with `git diff` + `patch` with suitable options.

Alternatively, we can also first merge `hpcgap/lib/helpbase.gi` into `lib/helpbase.gi`, at the cost of a couple of `atomic HELP_REGION do ... end` and `AtomicRecord` and `MigrateObj` calls being inserted...

Reply:
O.K.